### DISCUSSION TITLE
Aula 05 - Definindo o Contrato dos Comandos

### DISCUSSION THREAD

**Discussion by rikemorais** [balta-io/7196#29] [Perguntas e Respostas] [answered]
Teacher, olha eu mais uma vez. Sorry :/

Eu instalei o Flunt e conferi no XML do csproj:
`<PackageReference Include="flunt" Version="2.0.5" />`

Eu invoquei através do using:

`using Flunt.Validations;`

Mas o `IValidatable` está com um erro:

![image](https://user-images.githubusercontent.com/42439687/150677867-4716ffd4-e440-432d-925f-305e6e781bca.png)

_O nome do tipo ou do namespace "IValidatable" não pode ser encontrado (está faltando uma diretiva using ou uma referência de assembly?) 
[Gantting.Domain]csharp(CS0246)_

Eu percebi que o `using Flunt.Validations;` não está sendo usado, pois está com a mensagem "diretiva de uso desnecessária.

Eu vou tentar resolver sozinho, mas vou abrir aqui a new discussion caso mais alguém tenha essa dificuldade no futuro e se eu achar o problema eu posto aqui.

Att

**Comment by rikemorais** [comment, marked as answer]
Eu dei uma olhada no csproj do repositório do curso e vi que a versão do Flunt era a **1.0.4**. E quando eu instalei eu não especifiquei a versão. Então o terminal instalou a versão atual. Que é a **2.0.5** no momento em que escrevo.

Executei o `dotnet remove package flunt` dentro do todo.domain.

Depois eu adicionei o Flunt na versão do projeto do curso com o `dotnet add package flunt --version 1.0.4`. 

Tudo funcionou normalmente. 👯

**Reply by andrebaltieri** [reply]
Show @RikeMorais ❤️
